game/src/core/components/DrawComponent.java

```diff
@@ -149,6 +155,50 @@ public void currentAnimation(final IPath... animationName) {
         }
     }
 
+    /**
+     * Queue up an Animation to be considered as the next played Animation. Animations are given as
+     * an IPath Array, or multiple variables. Animation length is set as one frame. If you need to
+     * queue longer Animations use {@link #queueAnimation(int, IPath...)}
+     *
+     * @param next Array of IPaths to Animation
+     */
+    public void queueAnimation(IPath... next) {
+        queueAnimation(1, next);
+    }
+
+    /**
+     * Queue up an Animation to be considered as the next played Animation. Animations are given as
+     * an IPath Array, or multiple variables. Animation length is set to given parameter.
+     *
+     * @param forFrames number of Frames to play Animation for
+     * @param next Array of IPaths to Animation
+     */
+    public void queueAnimation(int forFrames, IPath... next) {
```

Comment:
Aus meiner sicht ist das jetzt eigentlich DIE Methode to use:

Sollten dann die "alten"/direkten Methoden zum setzne ` public void currentAnimation(final IPath... animationName)` etc. `private` gemacht werden (oder entfernt werden wenn die nicht mehr benutzt werden?)

Vielleicht könnte man dann auch hier das `queue` aus den Methodennamen werfen (auf der anderen Seite ist das ne gute Kommunikation nach außen)